com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -4,40 +4,67 @@
 
 namespace UnityEngine.Rendering.HighDefinition
 {
+    /// <summary>
+    /// Probe Volume Evaluation Mode.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ProbeVolumesEvaluationModes
     {
+        /// <summary>Probe volumes are disabled.</summary>
         Disabled = 0,
+        /// <summary>Probe volumes are evaluated in the light loop.</summary>
         LightLoop = 1,
+        /// <summary>Probe volumes are evaluated in the material pass.</summary>
         MaterialPass = 2,
     }
 
+    /// <summary>
+    /// Encoding of Probe Volumes.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ProbeVolumesEncodingModes
     {
+        /// <summary>Encoded using L0 spherical harmonics.</summary>
```

Comment:
Encodes probe volumes using L0 spherical harmonics.

Reply:
What are these spherical harmonics and what are the benefits of using different levels?

Reply:
Spherical Harmonics are a mathematical construct used to represent spherical functions. The more level the more precision you have (and the more costly, both in memory and perf).